### ISSUE TITLE
Some different findings during the competition

### ISSUE THREAD

**Issue by gathierry** [HiKapok/tf.fashionAI#1]
First of all, thanks for the sharing. 
I took part in this competition as well and I also used CPN. However, I got some findings different with yours during my experiments. Hope we can exchange some ideas.

> DetNet is better, perform almost the same as SEResNeXt, while SEResNet showed little improvement than ResNet

I also tried DetNet as backbone net but got a bad result. I guess it was because I trained it from scratch. One interesting is that, in my work, ResNet152 outperforms other backbone nets including ResNet-InceptionV2, SENet and NasNet.

> Enforce the loss of invisible keypoints to zero gave better performance

I tried both but didn't find a great difference here. So is there a significant improvement in your case?

> It's bad to do gaussian blur on the predicted heatmap

Still, no big difference for me, did you find it very worse?

> Ensemble of the heatmaps for fliped images is worser than emsemble of the predictions of fliped images

I got an opposite result for this. Maybe it depends on the model.

**Comment by HiKapok** [comment]
@gathierry thank you. I'm also very looking forward to learn from other top solutions and got more insights about this challange. 
1. In my opinion DetNet is better because of its larger top feature maps, I also used pre-trained resnet 
 weights to initialize the bottom layers of DetNet. I think the start layer when you generate FPN also matters. SE-Net had almost the same performance on my side, I have no idea about this. But SE-NeXt improved a lot. I didn't try resnet152, from my experience, resnet152 had ignorable improvements compared with resnet101.
2. It's very important to deal with those invisible keypoint explicitly.
3. gaussian blur on the prediction did hurt the performance a lot
4. most of implementation use the average of fliped heatmap, I did similar at first but found doing ensemble from keypoints position directly gave a little better performance.
